internal/cmd/base/servers.go

```diff
@@ -134,6 +135,8 @@ type Server struct {
 }
 
 func NewServer(cmd *Command) *Server {
+	reg := prometheus.NewRegistry()
```

Comment:
Have you tested this solution, using a NewRegistery instead of the DefaultRegisterer?  I think the prometheus http handler relies on the default registerer being used.

Reply:
Ah you're right none of the http metrics appear